honeybadgermpc/hbavss_multi.py

```diff
@@ -249,24 +52,20 @@ def __init__(self, publicparams, privateparams, send, recv):
         (t, n, crs, participantids, participantkeys, dealerid, sid) = publicparams
         (secret, pid) = privateparams
         assert dealerid == pid, "HbAvssDealer called, but wrong pid"
-        (poly, polyhat, sharedkeys, shares, encryptedshares,
-         witnesses, encryptedwitnesses) = ([], [], {}, {}, {}, {}, {})
-        for i in range(t+1):
-            poly.append(ZR.rand())
-            polyhat.append(ZR.rand())
-        poly[0] = ZR(secret)
-        sk = ZR.rand()
+        (sharedkeys, shares, encryptedshares,
+         witnesses, encryptedwitnesses) = ({}, {}, {}, {}, {})
+        poly = polynomials_over(use_rust=True).random(t, secret)
```

Comment:
doesn't `polynomials_over` have a required first argument?

Reply:
I have removed it. I have also added an `avss_parallel` method in `HbAvssLight` which effectively replaces `hbavss_multi.py` and added a test case for the same.

This is also how I expect the batch interface to be.